static/css/main.css

```diff
@@ -14,12 +14,12 @@ body {
   margin: 0 auto;
 }
 
-@media only screen and (max-width: 993px) {
+@media only screen and (min-width: 1500px) {
```

Comment:
What's the reason you had to change these numbers? did the previous boundaries not work?

Reply:
I was just testing the width I think but forgot to change it back so I fixed it!